test/support/pages/settings_page.js

```diff
@@ -20,21 +20,6 @@ define(function (require) {
       return common.findTestSubject('settingsNav advanced').click();
     },
 
-    setAdvancedSettings: function setAdvancedSettings(propertyName, propertyValue) {
```

Comment:
it's a utility function, and it's replicated 5 lines below: https://github.com/spalger/kibana/blob/6f27183cee357991e763997e75d4c14a0e8075d0/test/support/pages/settings_page.js#L35